cellrank/tools/_utils.py

```diff
@@ -1150,3 +1150,156 @@ def _info_if_obs_keys_categorical_present(
             and warn_once
         ):
             break
+
+
+def _one_hot(n, cat: Optional[int] = None) -> np.ndarray:
+    """
+    One-hot encode cat to a vector of length n.
+
+    If cat is None, return a vector of zeros.
+    """
+    if cat is not None and cat >= n:
```

Comment:
I don't think this check is necessary - it just prettifies the `IndexError` we'd get if it wasn't there.

Reply:
right, will remove this.